packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -228,6 +226,7 @@ export const ChatList = ({
 
       if (state.status === 'server connection established' && onConnectionChanged) {
         onConnectionChanged(true);
+        chatListClient.tryLoadChatThreads().catch(e => chatListClient.raiseFatalError(e as Error));
```

Comment:
void in front?

Reply:
My understanding is the rule is generally to void only when there is no then/catch/etc.